Build fails on sitemap.go

### Describe the bug
Building WriteFreely from source (version 0.9) fails with following error:

```
go-bindata -pkg writefreely -ignore=\\.gitignore schema.sql sqlite.sql
go get -tags='sqlite' -d -v ./...
go get: warning: modules disabled by GO111MODULE=auto in GOPATH/src;
	ignoring go.mod;
	see 'go help modules'
github.com/mattn/go-sqlite3 (download)
cd cmd/writefreely; go build -ldflags="-X 'github.com/writeas/writefreely.softwareVer=`git describe --tags | cut -c 2-`'" -v -tags='sqlite'
github.com/mattn/go-sqlite3
github.com/writeas/writefreely/parse
github.com/writeas/writefreely
# github.com/writeas/writefreely
../../sitemap.go:23:22: not enough arguments in call to stm.NewSitemap
	have ()
	want (int)
../../sitemap.go:78:4: index must be non-negative integer constant
../../sitemap.go:78:4: cannot use p.Post.Slug.NullString.String (type string) as type []interface {} in array or slice literal
../../sitemap.go:79:4: index must be non-negative integer constant
../../sitemap.go:79:18: cannot use "weekly" (type string) as type []interface {} in array or slice literal
../../sitemap.go:80:4: index must be non-negative integer constant
../../sitemap.go:80:4: cannot use true (type bool) as type []interface {} in array or slice literal
../../sitemap.go:81:4: index must be non-negative integer constant
../../sitemap.go:81:4: cannot use p.Post.Updated (type time.Time) as type []interface {} in array or slice literal
../../sitemap.go:86:33: index must be non-negative integer constant
../../sitemap.go:86:33: too many errors
make: *** [build] Error 2
```

### Steps to reproduce (if necessary)
Steps to reproduce the behavior:
1. go get -d -u github.com/writeas/writefreely/cmd/writefreely
2. cd go/src/src/github.com/writeas/writefreely
3. make build

Building on CentOS 7.

### Expected behavior
Build succeeded and writefreely available in go/bin.

**Version or last commit**:
0.9

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,13 +1,15 @@
 # Build image
-FROM golang:1.11.2-alpine3.8 as build
+FROM golang:1.12-alpine as build
 
 RUN apk add --update nodejs nodejs-npm make g++ git sqlite-dev
 RUN npm install -g less less-plugin-clean-css
+RUN go get -u github.com/jteeuwen/go-bindata/...
 
 RUN mkdir -p /go/src/github.com/writeas/writefreely
 WORKDIR /go/src/github.com/writeas/writefreely
 COPY . .
 
+ENV GO111MODULE=on
 RUN make build \
  && make ui
 RUN mkdir /stage && \

--- a/Dockerfile
+++ b/Dockerfile
@@ -18,6 +18,7 @@ RUN mkdir /stage && \
       /go/src/github.com/writeas/writefreely/static \
       /go/src/github.com/writeas/writefreely/pages \
       /go/src/github.com/writeas/writefreely/keys \
+      /go/src/github.com/writeas/writefreely/cmd \
       /stage
 
 # Final image
@@ -31,4 +32,4 @@ VOLUME /go/keys
 EXPOSE 8080
 USER daemon
 
-CMD ["bin/writefreely"]
+ENTRYPOINT ["cmd/writefreely/writefreely"]

